### ISSUE TITLE
Misnamed wheel file name for Debugpy 1.8.0

### ISSUE THREAD

**Issue by brettcannon** [microsoft/debugpy#1393]
https://pypi.org/project/debugpy/1.8.0/#files lists the pure Python wheel as supporting `py2.py3`, but the project's `requires-python` metadata says it requires Python 3.8 or newer. The `py2` should probably not be there.

**Comment by int19h** [comment]
I don't believe this is something that we control. We just build a wheel with `setup.py --universal`, and that's the tags that end up being applied. I vaguely recall looking into it at some point in the past and I couldn't figure out a way to remove this tag. I guess we could have a separate manual rename step in the pipeline, but given that it correctly specifies the minimum Python version in metadata, does it actually affect anything as is?

**Comment by brettcannon** [comment]
> We just build a wheel with `setup.py --universal`

Calling `setup.py` directly is actually deprecated, so there's the first issue. 😉 You should be using something like `build` as that's how everyone else is going to build the source distribution.

> given that it correctly specifies the minimum Python version in metadata, does it actually affect anything as is?

It's misleading for anyone who happens to look at the wheel file without seeing the metadata by cracking open the file.
